### PR TITLE
[DRAFT/SAMPLE] Jennyf/log message test

### LINKED ISSUE #1847: [Feature Request] Integrate IIdentityLogger with Id.Web for AspNetCore to see Wilson logs
Scope:
AspNetCore only

Create a class that implements IIdentityLogger to publish Wilson logs to AspNetCore's ILogger. This is so that AspNetCore can onboard and will no longer have to write a EventListener to read Wilson logs.

Proposal is [here](https://microsoft-my.sharepoint-df.com/:w:/p/srangavajhul/ETms1FahTcpPhjmXSGmb89YBfFTQEkL0sEYKgr-78x6AhA?e=JTXW3i)

### PR DESCRIPTION
**Pull request by jennyf19**
@sruke I was just looking at different ways of formatting the logs. Also turned the ";" to "" for CorrelationId.

image

### CODE REVIEW

File: src/Microsoft.IdentityModel.Extensions.AspNetCore/IdentityModelLoggerAdapter.cs [hunk 1]
```diff
@@ -102,7 +102,7 @@ public void Log(LogEntry entry)
         private static string FormatEntry(LogEntry entry)
         {
             StringBuilder sb = new StringBuilder();
-            sb.Append(entry.CorrelationId).Append(";");
+            sb.Append(entry.CorrelationId).Append("");
             sb.Append(entry.Message);
```
Comment (sruke) [inline src/Microsoft.IdentityModel.Extensions.AspNetCore/IdentityModelLoggerAdapter.cs:106]: We could add a null check for CorrelationId something like -

```csharp
String.IsNullOrEmpty(entry.CorrelationId) ? "" : "CorrelationId: " + entry.CorrelationId
```

File: src/Microsoft.IdentityModel.Logging/LogHelper.cs [hunk 2]
```diff
@@ -49,25 +49,15 @@ public class LogHelper
...
-        /// <summary>
-        /// The log message that indicates the date.
-        /// </summary>
-        private static string _dateLogMessage = "Date: {0}. ";
-
         /// <summary>
         /// The log message that is shown when PII is off.
         /// </summary>
-        private static string _piiOffLogMessage = "PII (personally identifiable information) logging is currently turned off. Set IdentityModelEventSource.ShowPII to 'true' to view the full details of exceptions. ";
+        private const string PiiOffLogMessage = "PII logging is off. See https://aka.ms/wilson/pii for details. ";
```
Comment (sruke) [inline src/Microsoft.IdentityModel.Logging/LogHelper.cs:55]: off and on could be displayed in uppercase (OFF/ON) to highlight the value. Looks like we have a PII link we can use here - https://aka.ms/IdentityModel/PII

**Comment by jennyf19** [comment]
@sruke do you still need this PR? LMK.

**Comment by jennyf19** [comment]
not needed anymore.

**Pull request closed without merging**